### PR TITLE
Add unit test for null denovo variant reports

### PR DESCRIPTION
**Pull request by IvoTod**

### CODE REVIEW

File: src/app/variant-reports/variant-reports.component.spec.ts [hunk 1]
```diff
@@ -1,25 +1,329 @@
...
+    expect(component).toBeTruthy();
+  });
+
+  it('should not have denovo', () => {
+    expect(component).toBeTruthy();
+    expect(component.currentDenovoReport).toBeUndefined();
+  })
+});
+
+fdescribe('VariantReportsComponent Denovo', () => {
```
Comment (iordanivanov) [inline src/app/variant-reports/variant-reports.component.spec.ts]: Remove the ```fdescribe``` please :)

File: src/app/variant-reports/variant-reports.component.spec.ts [hunk 2]
```diff
@@ -1,25 +1,329 @@
...
+    }
+    return of(variantReport);
+  }
+
+  getDownloadLink(): string {
+    return '';
+  }
+}
+
+fdescribe('VariantReportsComponent', () => {
```
Comment (iordanivanov) [inline src/app/variant-reports/variant-reports.component.spec.ts]: Remove this ```fdescribe``` please
